crossplane.yaml

```diff
@@ -32,64 +32,11 @@ metadata:
MDgtLjgyMzQwNDIuNzMzNjYwOC0xLjQ0ODU4MTUgMC0uNDI2OTUwNC0uMTI2OTU1My0uNzc3NjU5Ni0uMzgxMDA3MS0xLjA2NzM3NTkxLS4yNTM3Njk2LS4yODk3MTYzMS0uNzMzMzc4Ny0uNTQ4OTM2MTctMS40MjQ3MjExLS43OTI5MDc4bC0yLjA0NTI1MDctLjY4NjE3MDIxYy0xLjAyOTc0OS0uMzUwNzA5MjItMS43OTE0ODExLS44NjkxNDg5NC0yLjI1Njk4NC0xLjU1NTMxOTE1LS40NjU1MDMtLjY3MDkyMTk5LS43MDU0NDg2LTEuNDE4MDg1MTEtLjcwNTQ0ODYtMi4yMTA5OTI5MSAwLS42NDA0MjU1My4xMjcwOTY0LTEuMjA0NjA5OTMuMzgxMDA3MS0xLjY5MjU1MzE5LjI1Mzc2OTYtLjQ4Nzk0MzI2LjU5MjQ1ODMtLjkxNDg5MzYyIDEuMDE1NjQyOC0xLjI1MDM1NDYxLjQyMzE4NDUtLjM1MDcwOTIyLjkwMjc5MzYtLjYwOTkyOTA4IDEuNDY3MDM5Ni0uNzkyOTA3OC41NjQyNDYtLjE4Mjk3ODcyIDEuMTU2NzA0My0uMjU5MjE5ODYgMS43NzczNzQ5LS4yNTkyMTk4Ni4zMTAxOTQzIDAgLjYzNDc3NjguMDE1MjQ4MjMuOTQ0OTcxLjA2MDk5MjkxLjMyNDU4MjUuMDQ1NzQ0NjguNjIwNjcwNi4xMDY3Mzc1OS45MTY4OTk4LjE2NzczMDUuMjgyMTIzLjA3NjI0MTEzLjU1MDEzOTguMTUyNDgyMjcuODA0MTkxNi4yNDM5NzE2My4yNTM3Njk2LjA5MTQ4OTM2LjQ1MTM5NjguMTgyOTc4NzIuNTkyNDU4My4yNzQ0NjgwOC4xOTc0ODYxLjEyMTk4NTgyLjMzODU0NzYuMjQzOTcxNjMuNDIzMTg0NS4zODEyMDU2OC4wODQ2MzY5LjEyMTk4NTgxLjEyNjk1NTMuMjg5NzE2MzEuMTI2OTU1My41MDMxOTE0OXYuNzE2NjY2NjZjMCAuMzIwMjEyNzctLjExMjg0OTIuNDg3OTQzMjYtLjMyNDQ0MTQuNDg3OTQzMjYtLjExMjg0OTIgMC0uMjk2MjI5Mi0uMDYwOTkyOS0uNTM2MDMzNy0uMTgyOTc4NzItLjgwNDA1MDYtLjM5NjQ1MzktMS43MDY4NDQyLS41OTQ2ODA4NS0yLjcwODM4MDgtLjU5NDY4MDg1LS44MDQwNTA2IDAtMS40Mzg4MjczLjEzNzIzNDA0LTEuODc2MTE4LjQyNjk1MDM1LS40MzcyOTA2LjI4OTcxNjMyLS42NjI5ODkuNzMxOTE0OS0uNjYyOTg5IDEuMzU3MDkyMiAwIC40MjY5NTAzNi4xNDEwNjE1Ljc5MjkwNzguNDIzMTg0NSAxLjA4MjYyNDEyLjI4MjEyMy4yODk3MTYzMS44MDQwNTA1LjU3OTQzMjYyIDEuNTUxNjc2NS44Mzg2NTI0OGwyLjAwMzA3MzMuNjg2MTcwMjFjMS4wMTU2NDI4LjM1MDcwOTIyIDEuNzQ5MTYyNi44Mzg2NTI0OCAyLjE4NjQ1MzIgMS40NjM4Mjk3OS40MzcyOTA3LjYyNTE3NzMuNjQ4ODgzIDEuMzQxODQzOTcuNjQ4ODgzIDIuMTM0NzUxNzUgMCAuNjU1NjczOC0uMTI2OTU1NCAxLjI1MDM1NDYtLjM2Njc1OTkgMS43Njg3OTQzLS4yNTM5MTA4LjUxODQzOTgtLjU5MjQ1ODQuOTc1ODg2Ni0xLjAyOTc0OSAxLjM0MTg0NC0uNDM3MjkwNy4zODEyMDU3LS45NTkyMTgyLjY1NTY3MzgtMS41NjU3ODI3Ljg1MzkwMDctLjYzNDc3NjcuMjEzNDc1Mi0xLjI5Nzc2NTguMzIwMjEyOC0yLjAxNzE3OTQuMzIwMjEyOHoiLz48cGF0aCBmaWxsPSIjRkY5OTAwIiBkPSJNNDkuODc1NTA3IDQxLjAwNTkzNzJjLTQuNzA1MTM2NyAzLjY3NDQwNTEtMTEuNTQxMTcxMiA1LjYyNTAxNTItMTcuNDE4ODczOCA1LjYyNTAxNTItOC4yMzc1NjQ2IDAtMTUuNjU5OTUzNjEtMy4yMjA3NzQ4LTIxLjI2NjA3Mzk4LTguNTczNjExOS0uNDQzNDg0MTYtLjQyMzM4ODItLjA0MjkwMzk4LS45OTc5ODY1LjQ4NjI0NTE0LS42NjUzMjQ0IDYuMDYzNzYyODUgMy43MTk3NjgyIDEzLjU0MzIxNDE0IDUuOTcyNzk4NCAyMS4yODAzNzUyNCA1Ljk3Mjc5ODQgNS4yMTk5ODQ2IDAgMTAuOTU0ODE2OS0xLjE0OTE5NjYgMTYuMjMyMDA2Ny0zLjUwODA3NC43ODY1NzMtLjM3ODAyNTIgMS40NTg3MzU0LjU0NDM1NjMuNjg2MzIwNyAxLjE0OTE5Njd6bTEuOTU5NDI0OS0yLjM1ODg3NzNjLS42MDA3OTg4LS44MTY1MzQ1LTMuOTc1NzY5LS4zOTMxNDYzLTUuNTA2MDExMS0uMTk2NTczMi0uNDU3Nzg1NS4wNjA0ODQxLS41MjkyOTIxLS4zNjI5MDQyLS4xMTQ0MTA2LS42ODA0NDUzIDIuNjg4NjQ5Ni0xLjk5NTk3MzIgNy4xMDc2MTY3LTEuNDIxMzc0OCA3LjYyMjYwNzYtLjc1NjA1MDUuNTE0ODQ3Ny42ODA0NDU0LS4xNDMwMTMzIDUuMzUyODM3MS0yLjY2MDE5IDcuNTkwNzQ2My0uMzg1OTkyOC4zNDc3ODMyLS43NTc4MjczLjE2NjMzMTEtLjU4NjM1NDQtLjI4NzI5OTEuNTcyMDUzMS0xLjQ5Njk3OTkgMS44NDUwMTQzLTQuODY4OTY0OCAxLjI0NDM1ODUtNS42NzAzNzgyeiIvPjwvZz48L3N2Zz4=
```

Comment:
what icon will be used for this package now?

Reply:
@jbw976 I have consulted with @hasheddan today and he confirmed that:
* we are good to remove all annotations that are not part of the xpkg spec at https://crossplane.io/docs/v1.9/reference/xpkg.html#object-annotations
* icon annotation is not used at the moment and we manually inject icons for some specific packages
@hasheddan please confirm that my understanding is correct